dist/placeholder-buster.txt

```diff
@@ -18,6 +18,10 @@ pixiv.net##.ads-top-info
 reddit.com##div.fePAzF:has(div[id^="sidebar-"])
 reddit.com###overlay-comment-atf
 
+# https://github.com/NanoAdblockerLab/NanoContrib/pull/8
```

Comment:
I'd prefer to keep these specific unless there are enough websites using them, let's say 10 unrelated domains.

Reply:
Ah, okay. Duly noted. I've changed them to domain-specific entries now.